src/js/components/bulkDownload/awards/filters/dateRange/TimePeriodFilter.jsx

```diff
@@ -185,7 +185,8 @@ export default class TimePeriodFilter extends React.Component {
 
         if (this.state.showError) {
             errorDetails = (<DateRangeError
-                header={this.state.header} message={this.state.errorMessage} />);
+                header={this.state.header}
```

Comment:
Are we moving the start of all of these to new lines, after the opening parenthesis?

Reply:
There is a limit of 1 prop per line (and frankly this should have failed the old linter since it's multiline).
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-max-props-per-line.md